### PR TITLE
Renamed keep_dims to keep_axis, also accept None for full reduction.

### PR DESCRIPTION
**Pull request by david-berthelot**

### CODE REVIEW

File: objax/functional/loss.py [hunk 1]
```diff
@@ -65,35 +65,35 @@ def l2(x: JaxArray) -> JaxArray:
...
 
-def mean_absolute_error(x: JaxArray, y: JaxArray, keep_dims: Iterable[int] = (0,)) -> JaxArray:
+def mean_absolute_error(x: JaxArray, y: JaxArray, keep_axis: Optional[Iterable[int]] = (0,)) -> JaxArray:
     """Computes the mean absolute error between x and y.
     
     Args:
         x: a tensor of shape (d0, .. dN-1).
         y: a tensor of shape (d0, .. dN-1).
-        keep_dims: a sequence of the dimensions to keep.
+        keep_axis: a sequence of the dimensions to keep, use `None` to return a single value.
```
Comment (kihyuks) [inline objax/functional/loss.py:74]: a single value -> a scalar value

File: objax/functional/loss.py [hunk 2]
```diff
@@ -65,35 +65,35 @@ def l2(x: JaxArray) -> JaxArray:
...
 
-def mean_squared_error(x: JaxArray, y: JaxArray, keep_dims: Iterable[int] = (0,)) -> JaxArray:
+def mean_squared_error(x: JaxArray, y: JaxArray, keep_axis: Optional[Iterable[int]] = (0,)) -> JaxArray:
     """Computes the mean squared error between x and y.
     
     Args:
         x: a tensor of shape (d0, .. dN-1).
         y: a tensor of shape (d0, .. dN-1).
-        keep_dims: a sequence of the dimensions to keep.
+        keep_axis: a sequence of the dimensions to keep, use `None` to return a single value.
```
Comment (kihyuks) [inline objax/functional/loss.py:90]: a single value -> a scalar value

**Comment by kihyuks** [comment]
Added a comment for docstring. Besides, looks good to me.

File: objax/functional/loss.py (hunk 1 above)
Comment (david-berthelot) [reply objax/functional/loss.py]: Done

File: objax/functional/loss.py (hunk 2 above)
Comment (david-berthelot) [reply objax/functional/loss.py]: Done

**Pull request merged**